src/components/refactor/dialogs/two-windings-transformer-creation/phase-tap-changer-pane/phase-tap-changer-pane-utils.js

```diff
@@ -111,7 +111,7 @@ const phaseTapChangerValidationSchema = (id) => ({
                         .required()
                         .test(
                             'incoherentHighTapPosition',
-                            'InoherentHighTapPositionError',
```

Comment:
Could you add in the squash message that you fix this typo for translation please ?